.github/actions/run-smoke-tests-windows/windows-smoke-tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+
+$AMPLIFY_PATH version
+
+random_guid=$(powershell -Command "[guid]::NewGuid().ToString()")
+random_guid=${random_guid//-/}
+random_guid=${random_guid:0:6}
+projDir="smoke-test-${random_guid}"
+echo "AMPLIFY_PROJ_DIR=$projDir" >> $GITHUB_ENV
+
+mkdir $projDir
+cd $projDir
+
+$AMPLIFY_PATH init --yes
+
+$AMPLIFY_PATH status
+
+ADD_AUTH_REQUEST=$(cat ../.github/actions/run-smoke-tests-windows/add_auth_request.json)
```

Comment:
what path does this script expect to be run from? Could we pass in a base path as an arg to avoid confusion if we change the directory structure of these actions / scripts?